source/jormungandr/jormungandr/interfaces/v1/Schedules.py

```diff
@@ -231,7 +233,7 @@ class RouteSchedules(Schedules):
     def __init__(self):
         super(RouteSchedules, self).__init__("route_schedules")
```

Comment:
you need to define here `output_type_serializer`

and isn't it possible to use not `@get_serializer` but `get_obj_serializer` ?